Classes/Sizes/ContentElement.php

```diff
@@ -80,4 +98,39 @@ public function getParent(): ?ContentElementInterface
     {
         return $this->parent;
     }
+
+    public function getSizes(): array
+    {
+        return $this->sizes;
+    }
+
+    public function getMultiplier(): array
+    {
+        return $this->multiplier;
+    }
+
+    public function readConfiguration(): void
+    {
+        if (get_class($this) !== self::class) {
+            return;
+        }
+
+        $configurationPath = implode('.', [
+            'contentelements',
+            $this->contentType,
+            'image',
```

Comment:
We can't search for image here. This needs to be configurable. We already define which field should be used in the DataProcessor. We need to use that here.